pytest_odoo.py

```diff
@@ -125,7 +141,8 @@ def _worker_db_name():
             os.system(f"dropdb {db_name} --if-exists")
             os.system(f"createdb -T {original_db_name} {db_name}")
```

Comment:
```suggestion
            subprocess.run(["dropdb", db_name, "--if-exists"], check=True)
            subprocess.run(["createdb", "-T", original_db_name, db_name], check=True)
```

Not related to this PR but, @lmignon just in case the db name is not compatible with the shell separators.

Reply:
good catch